scripts/scil_run_commit.py

```diff
@@ -80,8 +79,7 @@
 import numpy as np
 import nibabel as nib
 
-from scilpy.io.streamlines import (lazy_streamlines_count,
```

Comment:
Why is `lazy_streamlines_count` no longer imported ? Was it unused before ?

Reply:
I am setting up my new computer and my autopep8 settings had too much power I think.